Calendar/Calendar/DayControl.cs

```diff
@@ -72,6 +80,12 @@ private void DayControl_MouseDown(object sender, MouseEventArgs e)
             
         }
 
+
+        /// <summary>
+        /// Color change to light blue when clicked on the day.
```

Comment:
Zamiast clickled on the day napisałbym when day is selected. Ale jest okej twoja decyzja.